src/chrome/content/rules/Apple.xml

```diff
@@ -103,27 +105,22 @@
 		- appleid
 		- appleseed
 		- appleseedcdn
-		- application
-		- as-images
 		- beta
 		- bugreport
 		- buyiphone
 		- buyiphone[1-4]
 		- certifications
```

Comment:
As long as they are listed as `<target host=` you can remove the `Fully covered hosts` block.